model/src/main/java/oracle/kubernetes/weblogic/domain/v2/DomainSpec.java

```diff
@@ -95,6 +98,19 @@
   @Expose
   private String logHome;
 
+  /**
+   * Whether the log home is enabled.
+   *
+   * @since 2.0
+   */
+  @SerializedName("logHomeEnabled")
+  @Expose
+  @Description(
+      "Specified whether the log home folder is enabled (Not required). "
+          + "Defaults to true if domainHomeInImage is false. "
+          + "Defaults to false if domainHomeInImage is true. ")
+  private Boolean logHomeEnabled; // Boolean object, null if unspecified
```

Comment:
Need to add new fields to equals(), hashCode(), and toString().

Reply:
Added those, ready for merge